Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/CaseEventListener.java

```diff
@@ -678,7 +676,8 @@ private OsAccountInstancesAddedTask(CentralRepository db, PropertyChangeEvent ev
 
         @Override
         public void run() {
-            if (!CentralRepository.isEnabled()) {
+            //Nothing to do here if the central repo is not enabled or the ingest is running and the setting to flag previously seen devices and users is not set to true
+            if (!CentralRepository.isEnabled() || (IngestManager.getInstance().isIngestRunning() && !IngestEventsListener.isFlagSeenDevices())) {
```

Comment:
So what if ingest is not running? I believe that we want to just go with whatever the IngestEventsListener currently believes, whether ingest is running or not.

Reply:
The decision to make them when ingest is not running is a decision based on previous decisions which were made on 'what to do until we have an artifacts pipeline. and was made for consistency,  I will link the relevant discussions in the related stories in JIRA.